v63003/u_inref/gtm8850.csh

```diff
@@ -16,7 +16,10 @@
 source $gtm_tst/com/gtm_test_setbgaccess.csh
 $gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
 echo "# Start background script that does MUPIP FREEZE -ONLINE -ON -NOAUTORELEASE or -AUTORELEASE"
-($SHELL $gtm_tst/$tst/u_inref/gtm8850freeze.csh & ; echo $!>>& bg.pid) >& f.out
+($SHELL $gtm_tst/$tst/u_inref/gtm8850freeze.csh & ; echo $!>>& bg.pid) >& f.outx
```

Comment:
Add a comment here that .outx is needed to prevent occasional OFRZAUTOREL error messages in f.out from being caught by test framework.